source/api/cypress-api/env.md

```diff
@@ -99,9 +95,9 @@ Remember, any changes that you make to environment variables using this API will
 ```
 
 ```javascript
-Cypress.env("host", "http://server.dev.local")
+Cypress.env('host', 'http://server.dev.local')
 
-Cypress.env("host") // => http://server.dev.local
+Cypress.env('host') // => http://server.dev.local
```

Comment:
A couple of the examples used double quotes: the Cypress doc style guide mandates single quotes so I corrected the errant examples.